tensorboard/components/tf_paginated_view/tf-paginated-view2.html

```diff
@@ -346,13 +361,17 @@
     },
 
     _insertItem(item, atIndex) {
+      if (!this._ensureTemplatized()) {
+        throw new Error('Expected templatized before inserting an item');
+      }
+
       let fragOrEl;
       const key = this.getItemKey(item);
       if (this._lruCachedItems.has(key)) {
         fragOrEl = this._lruCachedItems.get(key);
         this._lruCachedItems.delete(key);
       } else {
-        const inst = this.stamp({[this.as]: item});
+        const inst = new this.__ctor({[this.as]: item});
```

Comment:
Above you use `this._ctor`, but here you use `this.__ctor`. This seems
confusing. What’s the difference between the two?